File: tests/python/cases/case_117_dead_branch_global_02.py
"""
Task case 117: dead_branch_global_02 false-positive/false-negative stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_117_dead_branch_global_02(input_data):
    task_name = 'dead_branch_global_02'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'stress',
        'stage': 'evaluation',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    if False:
        RETAINED_CACHE['case_117'] = payload
    # SAFE: sink exists only in dead branch and never executes.
    return payload['checksum']

File: tests/go/cases/case_117_dead_branch_global_02.go
package escape_tests

var retainedCase117 = []map[string]string{}

func Case117DeadBranchGlobal02(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_branch_global_02",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "dead_branch_global_02:" + raw,
	}
	if false {
		retainedCase117 = append(retainedCase117, payload)
	}
	// SAFE: dead branch retention.
	return payload["checksum"]
}
